{
  "active_lock_reason": null,
  "assignee": {
    "avatar_url": "https://avatars3.githubusercontent.com/u/3115?v=4",
    "events_url": "https://api.github.com/users/mml/events{/privacy}",
    "followers_url": "https://api.github.com/users/mml/followers",
    "following_url": "https://api.github.com/users/mml/following{/other_user}",
    "gists_url": "https://api.github.com/users/mml/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/mml",
    "id": 3115,
    "login": "mml",
    "node_id": "MDQ6VXNlcjMxMTU=",
    "organizations_url": "https://api.github.com/users/mml/orgs",
    "received_events_url": "https://api.github.com/users/mml/received_events",
    "repos_url": "https://api.github.com/users/mml/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/mml/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/mml/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/mml"
  },
  "assignees": [
    {
      "avatar_url": "https://avatars3.githubusercontent.com/u/3115?v=4",
      "events_url": "https://api.github.com/users/mml/events{/privacy}",
      "followers_url": "https://api.github.com/users/mml/followers",
      "following_url": "https://api.github.com/users/mml/following{/other_user}",
      "gists_url": "https://api.github.com/users/mml/gists{/gist_id}",
      "gravatar_id": "",
      "html_url": "https://github.com/mml",
      "id": 3115,
      "login": "mml",
      "node_id": "MDQ6VXNlcjMxMTU=",
      "organizations_url": "https://api.github.com/users/mml/orgs",
      "received_events_url": "https://api.github.com/users/mml/received_events",
      "repos_url": "https://api.github.com/users/mml/repos",
      "site_admin": false,
      "starred_url": "https://api.github.com/users/mml/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/mml/subscriptions",
      "type": "User",
      "url": "https://api.github.com/users/mml"
    },
    {
      "avatar_url": "https://avatars3.githubusercontent.com/u/8061296?v=4",
      "events_url": "https://api.github.com/users/ixdy/events{/privacy}",
      "followers_url": "https://api.github.com/users/ixdy/followers",
      "following_url": "https://api.github.com/users/ixdy/following{/other_user}",
      "gists_url": "https://api.github.com/users/ixdy/gists{/gist_id}",
      "gravatar_id": "",
      "html_url": "https://github.com/ixdy",
      "id": 8061296,
      "login": "ixdy",
      "node_id": "MDQ6VXNlcjgwNjEyOTY=",
      "organizations_url": "https://api.github.com/users/ixdy/orgs",
      "received_events_url": "https://api.github.com/users/ixdy/received_events",
      "repos_url": "https://api.github.com/users/ixdy/repos",
      "site_admin": false,
      "starred_url": "https://api.github.com/users/ixdy/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ixdy/subscriptions",
      "type": "User",
      "url": "https://api.github.com/users/ixdy"
    }
  ],
  "author_association": "CONTRIBUTOR",
  "body": "https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/kubernetes-e2e-gke/17639 shows us trying to kube-down at the start and then blowing up when the cluster is not there.  No bueno.\n\n```\nAll components are up to date.\n... in gke:kube-down()\n... in gke:detect-project()\nERROR: (gcloud.container.clusters.delete) Some requests did not succeed:\n - args: ['ResponseError: code=404, message=The resource \"projects/k8s-jkns-e2e-gke-ci/zones/us-central1-f/clusters/jenkins-e2e\" was not found.\\nNo cluster named \\'jenkins-e2e\\' in k8s-jkns-e2e-gke-ci.']\n   exit_code: 1\n   message: ResponseError: code=404, message=The resource \"projects/k8s-jkns-e2e-gke-ci/zones/us-central1-f/clusters/jenkins-e2e\" was not found.\nNo cluster named 'jenkins-e2e' in k8s-jkns-e2e-gke-ci.\n```\n",
  "closed_at": "2016-10-13T20:29:38Z",
  "closed_by": {
    "avatar_url": "https://avatars3.githubusercontent.com/u/3115?v=4",
    "events_url": "https://api.github.com/users/mml/events{/privacy}",
    "followers_url": "https://api.github.com/users/mml/followers",
    "following_url": "https://api.github.com/users/mml/following{/other_user}",
    "gists_url": "https://api.github.com/users/mml/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/mml",
    "id": 3115,
    "login": "mml",
    "node_id": "MDQ6VXNlcjMxMTU=",
    "organizations_url": "https://api.github.com/users/mml/orgs",
    "received_events_url": "https://api.github.com/users/mml/received_events",
    "repos_url": "https://api.github.com/users/mml/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/mml/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/mml/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/mml"
  },
  "comments": 7,
  "comments_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/34747/comments",
  "created_at": "2016-10-13T18:55:46Z",
  "events_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/34747/events",
  "html_url": "https://github.com/kubernetes/kubernetes/issues/34747",
  "id": 182871183,
  "labels": [
    {
      "color": "f7c6c7",
      "default": false,
      "description": "Categorizes issue or PR as related to a flaky test.",
      "id": 264749912,
      "name": "kind/flake",
      "node_id": "MDU6TGFiZWwyNjQ3NDk5MTI=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/kind/flake"
    },
    {
      "color": "e11d21",
      "default": false,
      "description": "Highest priority. Must be actively worked on as someone's top priority right now.",
      "id": 114528068,
      "name": "priority/critical-urgent",
      "node_id": "MDU6TGFiZWwxMTQ1MjgwNjg=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/priority/critical-urgent"
    },
    {
      "color": "d2b48c",
      "default": false,
      "description": "Categorizes an issue or PR as relevant to SIG Cluster Lifecycle.",
      "id": 173494222,
      "name": "sig/cluster-lifecycle",
      "node_id": "MDU6TGFiZWwxNzM0OTQyMjI=",
      "url": "https://api.github.com/repos/kubernetes/kubernetes/labels/sig/cluster-lifecycle"
    }
  ],
  "labels_url": "https://api.github.com/repos/kubernetes/kubernetes/issues/34747/labels{/name}",
  "locked": false,
  "milestone": null,
  "node_id": "MDU6SXNzdWUxODI4NzExODM=",
  "number": 34747,
  "performed_via_github_app": null,
  "repository_url": "https://api.github.com/repos/kubernetes/kubernetes",
  "state": "closed",
  "title": "gke kube-down error blows up whole test run :'(",
  "updated_at": "2016-10-13T20:29:38Z",
  "url": "https://api.github.com/repos/kubernetes/kubernetes/issues/34747",
  "user": {
    "avatar_url": "https://avatars3.githubusercontent.com/u/3115?v=4",
    "events_url": "https://api.github.com/users/mml/events{/privacy}",
    "followers_url": "https://api.github.com/users/mml/followers",
    "following_url": "https://api.github.com/users/mml/following{/other_user}",
    "gists_url": "https://api.github.com/users/mml/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/mml",
    "id": 3115,
    "login": "mml",
    "node_id": "MDQ6VXNlcjMxMTU=",
    "organizations_url": "https://api.github.com/users/mml/orgs",
    "received_events_url": "https://api.github.com/users/mml/received_events",
    "repos_url": "https://api.github.com/users/mml/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/mml/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/mml/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/mml"
  }
}